boards/arm64/mimx93_evk/mimx93_evk_a55_sof.dts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 NXP
```

Comment:
We need more details here in the commit message about why do we need to introduce a new board and what's the general setup of running SOF.